data-structures/ts/queues-1.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import Queue from './lib/queue';
+
+export function getFront<T>(queue: Queue<T>): T | undefined {
+  if (queue.peek() === undefined) return undefined;
```

Comment:
This check isn't actually needed because line 6 below will return undefined for you if queue.peek() is undefined.